suites/pacific/rbd/tier-2_rbd_regression.yaml

```diff
@@ -86,3 +87,10 @@ tests:
       module: rbd_clonev2.py
       name: clones creation with v2clone format
       polarion-id: CEPH-83573308, CEPH-83573307, CEPH-83573309
```

Comment:
Please combine the tests and have one polarion id, we can discuss further if needed.

Reply:
done

Reply:
CEPH-83573309 - This test covers all functionality of V2clone and has steps of CEPH-883573308 CEPH-83573307